packages/grafana-ui/src/components/PieChart/PieChart.tsx

```diff
@@ -143,25 +143,17 @@ function useSliceHighlightState() {
   const { eventBus } = usePanelContext();
 
   useEffect(() => {
-    if (!eventBus) {
-      return;
-    }
-
     const setHighlightedSlice = (event: DataHoverEvent) => {
-      if (eventBus.isOwnEvent(event)) {
-        setHighlightedTitle(event.payload.dataId);
-      }
+      setHighlightedTitle(event.payload.dataId);
     };
 
     const resetHighlightedSlice = (event: DataHoverClearEvent) => {
-      if (eventBus.isOwnEvent(event)) {
-        setHighlightedTitle(undefined);
-      }
+      setHighlightedTitle(undefined);
     };
 
-    const subs = new Subscription()
-      .add(eventBus.subscribe(DataHoverEvent, setHighlightedSlice))
-      .add(eventBus.subscribe(DataHoverClearEvent, resetHighlightedSlice));
+    const subs = new Subscription();
+    subs.add(eventBus.getStream(DataHoverEvent).subscribe({ next: setHighlightedSlice }));
+    subs.add(eventBus.getStream(DataHoverClearEvent).subscribe({ next: resetHighlightedSlice }));
 
```

Comment:
^^^^ the `getStream()` flavor is a bit longer, but I think having a single interface is better than two options to do the same thing

Reply:
yea, but 99% can use the easier to use & more readable subscribe method. getStream is very rarely used. So to aid readability and consumption from users who are not familiar with Observable I think it might be worth it to have two signatures / methods of subscribing

Reply:
For the majority of use cases I would prefer the `subscribe` interface as it doesn't require knowing the specifics of the underlying arch.